### DISCUSSION TITLE
各位大佬，我想问问，我如何可以默认 模型

### DISCUSSION THREAD

**Discussion by hieivip** [lobehub/lobehub#1438] [General Question] [answered]
项目运行时候的模型是 openai，我想默认成google模型，请问怎么操作呢？

**Comment by arvinxx** [comment, marked as answer]
配置 [DefaultAgentConfig](https://chat-docs.lobehub.com/zh/self-hosting/environment-variables/basic#default_agent_config) 环境变量
